### ISSUE TITLE
multiplied entries in {{dolist>?status=undone}} 

### ISSUE THREAD

**Issue by lorenzsch** [cosmocode/do#25]
for some reason there are some to-does which are multiplied in the {{dolist>?status=undone}} .
am trying to identify the root cause but could not find it.
is there way to recreate the dolist from scratch?

hope you can help.

PS: am using dokuwiki on a stick for myself only since about one year and wanted to use the do-plugin heavily. it worked well quite a while and I got to like dokuwiki - even wanted to promote it and tell a friend to use it in his company but am now getting these double and tripple entries. I therefore could not recommend dokuwiki (yet).
maybe because I have changed a folder name in the folder hierarchy?

kind regards
Lorenz

**Comment by lorenzsch** [comment]
issue is solved. (it was a "user" know how issue ;-))

I had some records in the do.sqlite DB with capital letters and due to dokuwiki creating the namespace names only in lower-case the do.sqlite DB got confused. by removing the records in capital letters from do.sqlite (with SQLite) the problem was resolved. however, I lost some status records as I did not work too precise... but in my case this was no problem.
